CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Return with an error code when the `plugins validate` command is run and plugins have errors. Disable this behavior with `--no-fail-on-error` option. Also adds `--fail-on-warning` option to return with an error code when plugins have warnings. [#463](https://github.com/zowe/imperative/issues/463)
```

Comment:
Should the changelog be updated since `--fail-on-error` is no longer the default for zowe-v1-lts?

Reply:
Done. 😄